synapseclient/client.py

```diff
@@ -231,7 +231,7 @@ def login(*args, **kwargs):
         Using environment variable or `.synapseConfig`
 
             import synapseclient
-            syn = synapseclient.login()
+            syn = synapseclient.login(profile="user1)
```

Comment:
Rather than changing the first example, you should instead include a second example of logging in with a specified (non-default) profile since the base use case will still be the default behavior.